internal/nodes/avi_model_nodes_utils.go

```diff
@@ -121,6 +126,14 @@ func (o *AviVsNodeCommonFields) ConvertToRef() {
 		if o.ApplicationProfileRef != nil {
 			o.ApplicationProfileRef = proto.String("/api/applicationprofile?name=" + *o.ApplicationProfileRef)
 		}
+		SslKeyAndCertificateRefs := make([]string, 0, len(o.SslKeyAndCertificateRefs))
+		for i := range o.SslKeyAndCertificateRefs {
+			ref := fmt.Sprintf("/api/sslkeyandcertificate?name=" + o.SslKeyAndCertificateRefs[i])
```

Comment:
will this conversion called for all VS types? IF yes, `BuildL7Hostrule` already convert these objects to `/api/sslkeyandcertificate` format. Please check this.

Reply:
This method will only be invoked for L4 type VS. I will be invoked if the corresponding LoadBalancer service has L4Rule associated with it.